pootle/core/views.py

```diff
@@ -646,26 +660,26 @@ def get_context_data(self, *args, **kwargs):
 class PootleTranslateView(PootleDetailView):
     template_name = "editor/main.html"
 
-    @property
-    def ctx_path(self):
-        return self.pootle_path
-
-    @property
-    def vfolder_pk(self):
-        return ""
+    @cached_property
+    def extracted_path(self):
+        for result in extract_path.send(self.__class__, view=self):
+            if result[1]:
+                return result[1]
+        return self.request_path, "", {}
 
     @property
-    def display_vfolder_priority(self):
-        return False
+    def resource_path(self):
+        return (
+            "%s%s"
+            % (self.extracted_path[1],
+               self.request_path[len(self.ctx_path):]))
```

Comment:
@unho the request_path is used to here do determine the resource_path (ie pootle_path without the tp part or /project(s) part) - in the case of vfolders it also uses the extracted_path[1] which is basically any additional path elements - in this case ``$virtualfoldername/``